### ISSUE TITLE
Error while installing hashcat on macos

### ISSUE THREAD

**Issue by bendriss** [Homebrew/homebrew-cask#66715]
#### General troubleshooting steps

- [ x] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [ x] I have retried my command with `--force`.
- [ x] I ran `brew update-reset && brew update` and retried my command.
- [ x] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [ x] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [ x] I made doubly sure this is not a [checksum does not match](https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/checksum_does_not_match_error.md) error.

#### Description of issue

When trying to install hashcat on MacOs I get the error

#### Command that failed

```
$ brew cask install hashcat

```

#### Output of command with `--force --verbose --debug`

```
$ brew cask install hashcat --force --verbose --debug
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 2 taps (caskroom/cask and homebrew/cask).
No changes to formulae.

Warning: Unexpected method 'appcas' called on Cask movist-pro.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
Warning: Unexpected method 'appcas' called on Cask movist-pro.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
Error: Cask 'hashcat' is unavailable: No Cask with this name exists.
/usr/local/Homebrew/Library/Homebrew/cask/cmd/abstract_command.rb:55:in `rescue in casks'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/abstract_command.rb:48:in `casks'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/install.rb:16:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/abstract_command.rb:36:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:93:in `run_command'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:159:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:124:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:9:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:102:in `<main>'
Error: Kernel.exit
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:163:in `exit'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:163:in `rescue in run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:147:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:124:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:9:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:102:in `<main>'

```

#### Output of `brew cask doctor`

```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Library/Frameworks/Python.framework/Versions/3.6/bin/python3-config
  /Library/Frameworks/Python.framework/Versions/3.6/bin/python3.6-config
  /Library/Frameworks/Python.framework/Versions/3.6/bin/python3.6m-config

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libpcre.0.dylib
  /usr/local/lib/libpcre.dylib
  /usr/local/lib/libpcrecpp.0.dylib
  /usr/local/lib/libpcrecpp.dylib
  /usr/local/lib/libpcreposix.0.dylib
  /usr/local/lib/libpcreposix.dylib

Warning: Unbrewed .la files were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected .la files:
  /usr/local/lib/libpcre.la
  /usr/local/lib/libpcrecpp.la
  /usr/local/lib/libpcreposix.la

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libpcre.a
  /usr/local/lib/libpcrecpp.a
  /usr/local/lib/libpcreposix.a
```

#### Output of `brew tap`

```
acrogenesis/macchanger
caskroom/cask
homebrew/cask
homebrew/core
```

**Comment by Cottser** [comment]
Should be fixed by #66716

**Comment by vitorgalvao** [comment]
Was already solved.
